src/test/java/seedu/address/logic/parser/SortCommandParserTest.java

```diff
@@ -30,22 +30,6 @@ public void parse_emptyString_throwsParseException() {
         assertThrows(ParseException.class, () -> parser.parse(""));
     }
 
-    /*
-    @Test
-    public void parse_buyer_success() {
-        for (String param : CommandUtil.ACCEPTABLE_BUYER_PARAMETER) {
-            try {
-                Comparator<Buyer> comparator = getBuyerComparator(param);
-                SortBuyerCommand expected = new SortBuyerCommand(comparator);
-                assertParseSuccess(parser, param, expected);
-            } catch (ParseException e) {
-                assert false;
-            }
-        }
-    }
-
-     */
-
```

Comment:
We do have one it's just hard to test because the comparator only returns true for if referring to the same object. I am trying to test it in SortCommandParserUtil.